### PR TITLE
Use vars for service name reference in commands

### LINKED ISSUE #2302: Consider using kustomize vars for manifests
Consider replacing container arguments which reference service with [kustomize's vars](https://kubectl.docs.kubernetes.io/pages/app_customization/config_reflection.html) to make it easy for deployments with `namePrefix` or `nameSuffix`.

### PR DESCRIPTION
**Pull request by imranismail**
Closes #2302

This allows Kustomize's `nameSuffix` and `namePrefix` transformer to work with the service name reference in the commands.

### CODE REVIEW

**Comment by alexec** [comment]
Probably need to check all combinations work.

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Organizations below are **officially** using Argo CD. Please send a PR with your
 1. [Yieldlab](https://www.yieldlab.de/)
 1. [UBIO](https://ub.io/)
 1. [Volvo Cars](https://www.volvocars.com/)
+1. [Fave](https://myfave.com)
+1. [127Labs](https://127labs.com/)
```
[the remark this answers is not in the document]
Comment (imranismail) [reply README.md]: Yep! It's my company :smile: 

I'll reorder the list.

**Comment by imranismail** [comment]
> Probably need to check all combinations work.

Is there an E2E test for this or must this be done manually?

**Comment by alexec** [comment]
No need to write a test. @jessesuen could you review - I think you know this Kustomize feature well.

**Review by jessesuen — APPROVED** [review]
Nice!

**Comment by jessesuen** [comment]
Please rebase on master and we can merge.

**Pull request merged**